static/js/Router.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ReactElement } from "react"
```

Comment:
What's the convention for using `tsx` vs `ts`? I see both used in this PR

Reply:
afaik `.tsx` tells typescript to use the JSX parser, and `.ts` is just the normal one. If you change a file with JSX in it to `.tsx` you'll get all kinds of syntax errors. If we wanted to standardize on one thing I suppose we could just use `.tsx` everywhere (there may also be a compiler option to always use the JSX mode regardless of the file extension)

Reply:
I'm fine with both, just curious when to use each one. It makes sense to leave it as is